### PR TITLE
Bibliographic and project review

### PR DESCRIPTION
**Pull request by jorgejesus**
- Improved list of scientific research paper
- New project section

### CODE REVIEW

File: projects/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+PyWPS has been used or supported by the following projects
+
+
+## iGUESS - Smart City Energy
+Smart City Energy supports cities in the development of innovative strategies and opportunities for sustainable energies transitions. 
+
+[http://iguess.tudor.lu/](http://iguess.tudor.lu/)
+
+
+## WMO WOUDC (World Ozone & Ultraviolet Radiation Data Centre)
```
Comment (tomkralidis) [inline projects/index.md:19]: s/&/and/

File: projects/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+World Meteorological Organization (WMO) data centre supporting the Global Atmosphere Watch (GAW) program operated by Environment and Climate Change Canada. )
```
Comment (tomkralidis) [inline projects/index.md:20]: Remove trailing bracket at end of sentence?

File: projects/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+## iGUESS - Smart City Energy
+Smart City Energy supports cities in the development of innovative strategies and opportunities for sustainable energies transitions. 
+
+[http://iguess.tudor.lu/](http://iguess.tudor.lu/)
+
+
+## WMO WOUDC (World Ozone & Ultraviolet Radiation Data Centre)
+World Meteorological Organization (WMO) data centre supporting the Global Atmosphere Watch (GAW) program operated by Environment and Climate Change Canada. ) 
+
+[http://www.opengeospatial.org/blog/2418](http://www.opengeospatial.org/blog/2418)
```
Comment (tomkralidis) [inline projects/index.md:22]: reverse the order of these two links.

**Comment by jachym** [comment]
Can we add http://openwps.apcc21.org ?

**Pull request merged**